code/modules/projectiles/gun.dm

```diff
@@ -1608,6 +1610,9 @@ not all weapons use normal magazines etc. load_into_chamber() itself is designed
 		user = gun_user
 
 	if(flags_gun_features & GUN_AMMO_COUNTER && current_mag)
+		// toggleable spam control.
+		if(flags_gun_features & GUN_LIMIT_AMMO_DISPLAY && current_mag.current_rounds % 5 != 0 && current_mag.current_rounds > 15)
```

Comment:
This is wonky with burst fire if the burst doesn't end on a multiple of 5, and though understandable you then start displaying per round when the mag is starting to go dry - it just feels like its randomly bugged out.

You should have this either send a message ever we *pass* the threshold, or just always on a burst. You could also convey that the preference only works when the mag count is above 15.